### PR TITLE
f-enhancement/gh-357-make-list-view-for-team-calendar

### PR DESCRIPTION
**Pull request by aberonja**
Created list calendar for team.

GH issue: #357

### CODE REVIEW

File: src/main/frontend/src/components/team-calendar/ListView.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    return (
+      <EmployeeContainer>
+        <InfiniteScroll
+          loadMore={page => {
+            this.props.requestUserDaysForTeam({
+              month: moment(this.props.selectedMonth, 'YYYY-MMMM').format(
+                'YYYY-MM'
+              ),
+              page:
+                this.props.userDaysForTeam.page + 1 !== page
```
Comment (iscekic) [inline src/main/frontend/src/components/team-calendar/ListView.js]: Leave a comment explaining why this is necessary.

File: src/main/frontend/src/pages/CreateYear.js [hunk 2]
```diff
@@ -92,6 +92,7 @@ class CreateYear extends Component {
   }
 }
 
+// TODO: Check if bindActionCreators is needed
```
Comment (iscekic) [inline src/main/frontend/src/pages/CreateYear.js]: Resolve this `TODO`.

File: src/main/frontend/src/pages/EditEmployee.js [hunk 3]
```diff
@@ -94,6 +94,7 @@ class EditEmployee extends Component {
             isSubmitting
           }) => (
             <React.Fragment>
+              {console.log(values)}
```
Comment (iscekic) [inline src/main/frontend/src/pages/EditEmployee.js]: Remove this line.

File: src/main/frontend/src/store/reducers/userDaysForTeamReducer.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+import {
+  receiveUserDaysForTeam,
+  removeUserDaysForTeamOnYearChange,
+  resetUserDaysValues,
+  updateUserDaysForTeam
+} from '../actions/teamActions';
+
+const handleUpdate = (results, userId, userDays) => {
+  const newResults = _cloneDeep(results);
+  const userArrayIndex = newResults.indexOf(
```
Comment (iscekic) [inline src/main/frontend/src/store/reducers/userDaysForTeamReducer.js]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex

File: src/main/frontend/src/store/reducers/userDaysForTeamReducer.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    return {
+      ...state,
+      results: handleUpdate(
+        state.results,
+        action.payload.userId,
+        action.payload.userDays
+      )
+    };
+  },
+  [removeUserDaysForTeamOnYearChange]: () => {
```
Comment (iscekic) [inline src/main/frontend/src/store/reducers/userDaysForTeamReducer.js]: Extract this.

File: src/main/frontend/src/styles/globalStyles.js [hunk 6]
```diff
@@ -85,6 +85,11 @@ input {
     }
 }
 
+InfiniteScroll {
+    height: 70vh;
```
Comment (iscekic) [inline src/main/frontend/src/styles/globalStyles.js]: Indentation.

File: src/main/frontend/src/components/team-calendar/ListView.js (hunk 1 above)
Comment (aberonja) [reply src/main/frontend/src/components/team-calendar/ListView.js]: I changed this part of code a bit and I left out the comment.

**Pull request merged**